main.go

```diff
@@ -138,6 +139,13 @@ func buildFactsSection(cfg config, buildSuccessful bool) Section {
 	}
 }
 
+// Builds a Section containing a list of Image
+func buildImagesSection(cfg config) Section {
+	section := Section{}
+	section.Images = []Image{cfg.SectionImage}
```

Comment:
Ah yes, sorry, that seems like a silly oversight now. I'll chalk it up to never having written Go before. I just downloaded JetBrains' Go IDE but couldn't figure out how to compile my changes to validate them. I'll get that sorted out, but I'm going out of town for three days so it will have to be the beginning of next week.

Reply:
Thanks for the pointers. I was able to edit the run configuration and add environment variables. I created a test Teams channel and was able to send a test message. I'll attach a screenshot so you can see what the resulting message looks like.